.github/workflows/csound_builds.yml

```diff
@@ -1,65 +1,84 @@
 name: csound_builds
 
-on: 
-  push:
-    branches:
-      - develop
-      - master
-    pull_request:
-      branches:
-        - develop
+on:
```

Comment:
Any particular reason to reformat this to 4-spaces?

Reply:
Sorry my autoformatter went nuts. I'll correct it now.